packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -309,8 +308,6 @@ export function reasonUserCantReceiveEmails(user: DbUser): string|null
 {
   if (!user.email)
     return "No email address";
-  if (!userEmailAddressIsVerified(user))
-    return "Address is not verified";
```

Comment:
By removing this reason for not sending emails, the EA Forum now will send emails to users without verified emails.